src/sources/splunk_hec.rs

```diff
@@ -40,6 +40,7 @@ pub struct SplunkConfig {
     address: SocketAddr,
     /// Splunk HEC token
     token: Option<String>,
```

Comment:
I think we should mark this one as deprecated and encourage users to use `valid_tokens`.